### DISCUSSION TITLE
NAS通过矿神源安装的alist，怎么取消2FA验证

### DISCUSSION THREAD

**Discussion by Qiao-Love** [AlistGo/alist#6355] [Q&A]
NAS通过矿神源安装的alist，然后2FA验证不小心删了，现在进不去后台的情况下怎么取消2FA验证

**Comment by anwen-anyi** [comment]
看文档使用命令删除
- https://alist.nn.ci/zh/guide/advanced/2fa.html

**Reply by Qiao-Love** [reply]
![Qiao_2024-04-16_14-01-36](https://github.com/alist-org/alist/assets/23590124/63475733-a049-4732-a165-7990cc62a347)
我这个好像不行

**Reply by anwen-anyi** [reply]
> ![Qiao_2024-04-16_14-01-36](https://private-user-images.githubusercontent.com/23590124/322719477-63475733-a049-4732-a165-7990cc62a347.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTMyNDg5OTgsIm5iZiI6MTcxMzI0ODY5OCwicGF0aCI6Ii8yMzU5MDEyNC8zMjI3MTk0NzctNjM0NzU3MzMtYTA0OS00NzMyLWExNjUtNzk5MGNjNjJhMzQ3LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA0MTYlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNDE2VDA2MjQ1OFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWE5YWFlMzNhMDc4YTVkZGNhZjNhMmE2ZGVkYTA1YjU5NmM0MTA0NmFmNzJmNDEwZDM5MDgxYzNiMWUxNmQ4NDYmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.0KOSNotRUA_k4-Lw39aT0CGxJ3JETgh1oOam0DMxlfo) 我这个好像不行

CD到AList目录 在执行 别直接在当前目录执行命令

**Reply by Qiao-Love** [reply]
![Qiao_2024-04-17_19-50-58](https://github.com/alist-org/alist/assets/23590124/ddea9b91-7148-410d-8b88-29de0949c25b)
这样吗哥，还是不行
![444D1AB4](https://github.com/alist-org/alist/assets/23590124/3870ee5c-5088-4ac6-b128-c65a537b5f32)

**Comment by seaowen** [comment]
老哥成了吗？我也这样，用命令关不了两步验证，提示[del_user_cache_online] error: token is invalidated
